Audit r vignettes (#686)
chanzuckerberg/cellxgene-census: api/r/cellxgene.census/vignettes_/census_dataset_presence.Rmd

The `cellxgene.census` R package contains a convenient API to open any version of the Census (by default, the newest stable version).

```{r}

> library("cellxgene.census")

Reviewer: I don't feel strongly about this, but the quotes aren't required and not usually used IME. (Throughout)

```suggestion
library(cellxgene.census)
```

Author: I think there's been a shift. Mainly because if the package is not installed you get an variable not found error instead of a package not installed error

- the first dimension of the presence matrix is the dataset's `soma_joinid`, as stored in the `census_datasets` dataframe.
- the second dimension of the presence matrix is the feature's `soma_joinid`, as stored in the `var` dataframe.

> The presence matrix has a method `$take()` that let's you slice it by the `soma_joinid` from dataset and var joinid. And the full presence matrix or slices of it can be then exported to a regular matrix with the method `$get_one_based_matrix()`

Reviewer: ```suggestion
The presence matrix has a method `$take()` that lets you slice it by `soma_joinid`s from `census_datasets` and `var`. And the full presence matrix, or slices of it, can then be exported to a regular matrix with the method `$get_one_based_matrix()`
```

Let's find out if the the gene `"ENSG00000286096"` was measured in the dataset with id `"97a17473-e2b1-4f31-a544-44a60773e2dd"`.

```{r}

> # Get somo_joinid for datasets and genes of interest

Reviewer: ```suggestion
# Get soma_joinid for datasets and genes of interest
```
